packages/KTable/KTable.vue

```diff
@@ -463,6 +468,10 @@ export default defineComponent({
     sortHandlerFn: {
       type: Function,
       default: () => ({})
+    },
+    paginationType: {
+      type: String,
+      default: 'default'
```

Comment:
We should add a type interface here for the `beta` branch port

Reply:
I've added this to my beta branch port 👍